### ISSUE TITLE
Consider alignment to existing standards

### ISSUE THREAD

**Issue by haydens30** [opengeospatial/ogcapi-routes#58]
Open Question: Have you previously sought alignment with other existing routing Open Standards, especially those already used in transport & mobility?.(E.g. GTFS "route" and "shapes" and The Open Transport Initiative's "journey" spec).

**Comment by haydens30** [comment]
E.g. 
GTFS : route & shape
https://developers.google.com/transit/gtfs/reference
OTI : usage
https://app.swaggerhub.com/apis/open-transport/customer-account/1.0.2#/developers/getUsage

**Comment by jeffharrison** [comment]
Thanks very much for your message. Would you be willing to brief the next meeting of OGC Routing SWG on your efforts? I think there would be interest...

Please note the SWG is still in the process of scheduling this next session, so we're flexible on dates and times...

Best Regards,
Jeff

**Comment by haydens30** [comment]
Certainly. Happy to help, share and move us all forward. Just let me know when the next meeting is.

**Comment by edpars0ns** [comment]
That would be very valuable @haydens30 thank you, this is an area with many existing standards as you mention GTFS but also DATEX and DATEX 2 for dynamic data..

**Comment by jeffharrison** [comment]
Group reviewed these on 8 Aug 2023.  OGC Routing SWG will request that SWG members review these over the next two weeks and send any comments to the email list or post on GitHub before 22 August 2023.

In addition, SWG Chair would like to invite @haydens30 to present these specifications at an upcoming OGC Routing SWG meeting (the next meeting is scheduled for 22 August 2023 at 1030 AM ET).

Thanks and Regards,
Jeff

**Comment by jeffharrison** [comment]
@haydens30 and I would like to apologize for the (very) delayed response on this ; - ) We hope to talk to you soon..

Thanks and Regards,
Jeff

**Comment by ghobona** [comment]
Figure 8 and Figure 32 of the document ['Future of Transport: System interoperability and standards'](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/936211/BSI-Future-of-Transport-system-interoperability-and-standards-report-accessible.pdf) present an illustration of the standards landscape.

**Comment by haydens30** [comment]
Yes, BSI made some pretty good recommendations 3 years ago.
Unfortunately none of them have been taken up by UK Gov yet.

**Comment by haydens30** [comment]
@jeffharrison and all - I think you need to be aware of this very recent development by MobilityData  - the org that manages the General Transit Feed Specification (GTFS)
https://github.com/google/transit/issues/391

**Comment by skyNacho** [comment]
This issue has been completed in https://github.com/opengeospatial/ogcapi-routes/pull/77
